### PR TITLE
Implemented encoder and demux

### PR DESCRIPTION
**Pull request by ghost**
Implemented Encoder and Demultiplexer class code for Review. Please note  I have implemented the logic of encoder and demux for review. Based on your reviews, I will correct the code.

### CODE REVIEW

**Comment by rajat974** [comment]
Already done ! See #61

**Comment by ghost** [comment]
@salil93 Could you Please review this.

File: BinPy/Combinational/combinational.py [hunk 1]
```diff
@@ -22,3 +32,72 @@ def setInputs(self,inputs):
 			return None
 
 		self.inputs = inputs[:]
+		
+
+"""
```
Comment (jayrambhia) [inline BinPy/Combinational/combinational.py:37]: Put these comments in the class

File: BinPy/Combinational/combinational.py [hunk 2]
```diff
@@ -22,3 +32,72 @@ def setInputs(self,inputs):
...
+ENCODER:
+input lines, output lines
+if output lines = n, input lines = 2 power n
+input lines must be a power of 2
+"""    
+class Encoder:
+
```
Comment (jayrambhia) [inline BinPy/Combinational/combinational.py:44]: Add docs. Proper docs. with example too.

``` python
"""
 Brief description of the class
 input params
 output params
 Example
"""
```

**Comment by jayrambhia** [comment]
And, also add some examples in the examples folder. Add tests too.

**Pull request closed without merging**